--- E01HelloWorldStubImpl.cpp.bak       2025-01-21 04:08:13.318976076 +0000
+++ E01HelloWorldStubImpl.cpp   2025-01-21 04:14:04.708505087 +0000
@@ -4,6 +4,11 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.

 #include "E01HelloWorldStubImpl.hpp"
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <cstring>
+#include <fstream>

 E01HelloWorldStubImpl::E01HelloWorldStubImpl() {
 }
@@ -15,6 +20,8 @@
         std::string _name, sayHelloReply_t _reply) {

     std::stringstream messageStream;
+    std::string command = "echo TEMP_ID = "+_name+">/home/cmd/tmp/debug.txt";
+    system(command.c_str());

     messageStream << "Hello " << _name << "!";
     std::cout << "sayHello('" << _name << "'): '" << messageStream.str() << "'\n";
